homeassistant/components/garmin_connect/alarm_util.py

```diff
@@ -0,0 +1,52 @@
+"""Utility method for converting Garmin Connect alarms to python datetime."""
+from datetime import date, datetime, timedelta
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+DAY_TO_NUMBER = {
+    "Mo": 1,
+    "M": 1,
+    "Tu": 2,
+    "We": 3,
+    "W": 3,
+    "Th": 4,
+    "Fr": 5,
+    "F": 5,
+    "Sa": 6,
+    "Su": 7,
+}
+
+
+def calculate_next_active_alarms(alarms):
+    """
+    Calculate garmin next active alarms from settings.
+
+    Alarms are sorted by time
+    """
+    active_alarms = []
+    _LOGGER.debug(alarms)
+
+    for alarm_setting in alarms:
+        if alarm_setting["alarmMode"] != "ON":
+            continue
+        for day in alarm_setting["alarmDays"]:
+            if day == "ONCE":
+                midnight = datetime.combine(date.today(), datetime.min.time())
+                alarmtime = alarm_setting["alarmTime"]
```

Comment:
Please use snake_case for all variable names and move this outside of the if...else condition since it is used in both cases.
```suggestion
                alarm_time = alarm_setting["alarmTime"]
```

Reply:
done, thanks